prow/plugins/label/label.go

```diff
@@ -211,6 +217,12 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		if err := gc.AddLabel(ae.org, ae.repo, ae.number, sigLabel); err != nil {
 			log.WithError(err).Errorf("Github failed to add the following label: %s", sigLabel)
 		}
+
+		if kLabel, ok := kindMap[kind]; ok {
```

Comment:
Your regex group matches `misc|test-failures|bugs|feature-requests|proposals|pr-reviews|api-reviews`, and then you look in a map for `bugs, feature-requests, proposals`?

Should you make the map only contain the "alias", and if not in the alias map, just apply `kind/$match`? And you should also maybe allow people not to use the alias.